app/models/site.rb

```diff
@@ -4,6 +4,7 @@ class Site < ApplicationRecord
   include PostcodeNormalize
   include RegionCode
   include TouchProvider
+  include ActiveModel::Validations
```

Comment:
This is already included by `ActiveRecord`

Reply:
thanks, amended